lib_test/query-pci.json

```diff
@@ -0,0 +1 @@
+{ "execute": "query-pci" }
```

Comment:
I think we might have to add a test to check that we can parse the reply, but that can be done as a separate PR.